### DISCUSSION TITLE
Inquiry Regarding ElastAlert2 Compatibility with Elasticsearch and Kibana Version 8.8.2

### DISCUSSION THREAD

**Discussion by jungyoo601** [jertel/elastalert2#1567] [Q&A]
Hello Manager,

I would like to use ElastAlert2, and I have some questions about it.

I’m currently using ES and Kibana version 8.8.2, but when I checked the setup.py file, it specifies elasticsearch==7.10.1.

Is there any way I can use ElastAlert2 in my current setup?

**Comment by nsano-rururu** [comment]
Supported

look
https://github.com/jertel/elastalert2/issues/92#issuecomment-928217013

**Reply by nsano-rururu** [reply]
When it comes to ES8 support, we are doing detailed reviews visually and by checking the operation. There is also a branch in the internal logic between ES version 7 and 8.
https://github.com/jertel/elastalert2/pull/744/files

**Reply by jungyoo601** [reply]
Thank you very much for your response. It was very helpful.
